packages/common/src/groups/HubGroups.ts

```diff
@@ -142,12 +147,14 @@ export async function createHubGroup(
   );
 
   // create or update entity settings
-  const entitySetting = await createOrUpdateEntitySettings(
-    { ...hubGroup, id: entity.id },
-    context.hubRequestOptions
-  );
-  entity.entitySettingsId = entitySetting.id;
-  entity.discussionSettings = entitySetting.settings.discussions;
+  if (context.isPortal) {
```

Comment:
This needs to be `if (!context.isPortal) {` otherwise it'll only fetch the settings in ES.